entrypoint.sh

```diff
@@ -38,7 +38,9 @@ fi
 # This possibly changes the log directory that might be mounted in
 if check_set "${ENABLE_AIRFLOW_CHOWN}"; then
   echo "Chowning ${AIRFLOW_HOME} to ${AIRFLOW_USER}:${AIRFLOW_GROUP}..."
-  chown "${AIRFLOW_USER}:${AIRFLOW_GROUP}" -R "${AIRFLOW_HOME}/"
+  mkdir -pv "${AIRFLOW_HOME}/dags"; \
+  mkdir -pv "${AIRFLOW_HOME}/logs"; \
```

Comment:
We don't need these anymore right?

Best not to manually create, because users might override the env vars to change the logs dir?

Reply:
Ref:
- https://airflow.apache.org/docs/apache-airflow/1.10.12/configurations-ref.html#dags-folder
- https://airflow.apache.org/docs/apache-airflow/1.10.12/configurations-ref.html#base-log-folder

Reply:
Discussed on Slack and the consensus is that if these directories are auto-created, they will not belong to `root` group.